src/video2x-qt6_ja_JP.ts

```diff
@@ -1,742 +1,743 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
 <TS version="2.1" language="ja_JP">
-<context>
+  <context>
     <name>AboutDialog</name>
     <message>
-        <location filename="aboutdialog.ui" line="20"/>
-        <source>About Video2X Qt6</source>
-        <translation>Video2X Qt6 について</translation>
+      <location filename="aboutdialog.ui" line="20" />
+      <source>About Video2X Qt6</source>
+      <translation>Video2X Qt6 について</translation>
     </message>
-</context>
-<context>
+  </context>
+  <context>
     <name>MainWindow</name>
     <message>
-        <location filename="mainwindow.ui" line="212"/>
-        <location filename="mainwindow.cpp" line="363"/>
-        <source>Progress</source>
-        <translation>進行</translation>
+      <location filename="mainwindow.ui" line="212" />
+      <location filename="mainwindow.cpp" line="363" />
+      <source>Progress</source>
+      <translation>進行</translation>
     </message>
     <message>
-        <location filename="mainwindow.ui" line="267"/>
-        <source>Frames/s:</source>
-        <translation>フレーム/秒：</translation>
+      <location filename="mainwindow.ui" line="267" />
+      <source>Frames/s:</source>
+      <translation>フレーム/秒:</translation>
```

Comment:
Why where the colons here changed from CJK to Latin?

Reply:
I use Latin for my text. I've standardized it.

Reply:
Shouldn't we be using the punctuation of the translated language instead of Latin?

Reply:
Double colons, semicolons and parentheses can look messy in the UI.

Reply:
I took a look at [Microsoft's localization style guides](https://learn.microsoft.com/en-us/globalization/reference/microsoft-style-guides). I'm still under the impression that it is better to always use the punctuation of the target language rather than Latin. I will undo the changes for consistency.